swift/Samples/TicTacToe/Sources/Game/RunGameWorkflow.swift

```diff
@@ -84,19 +84,19 @@ extension RunGameWorkflow {
 // MARK: Rendering
 
 extension RunGameWorkflow {
-    typealias Rendering = AlertContainerScreen<ModalContainerScreen<BackStackScreen>>
+    typealias Rendering = AlertContainerScreen<ModalContainerScreen<BackStackScreen<AnyScreen>>>
 
     func render(state: RunGameWorkflow.State, context: RenderContext<RunGameWorkflow>) -> Rendering {
         let sink = context.makeSink(of: Action.self)
         var modals: [ModalContainerScreenModal] = []
         var alert: Alert?
 
-        var backStackItems: [BackStackScreen.Item] = [BackStackScreen.Item(
-            screen: newGameScreen(
+        var backStackItems: [BackStackScreen<AnyScreen>.Item] = [BackStackScreen.Item(
+            screen: AnyScreen(newGameScreen(
```

Comment:
This probably warrants a `asAnyScreen()` helper on `Screen`. The code here would then become:

```swift
screen: newGameScreen(
    sink: sink,
    playerX: state.playerX,
    playerO: state.playerO
).asAnyScreen(),
```

Reply:
Or I guess we could add an extension:

```swift
extension BackStackScreen where Content == AnyScreen {
    init<S: Screen>(…) {
        // Do the wrapping into AnyScreen here
    }
}
```